devops/ansible/playbook.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Install Nvidia Docker
+  hosts: all
+  become: yes
+  vars:
+    viame_bundle_url: https://viame.kitware.com/api/v1/item/60519746bc3f8f99bba1a5f6/download
+    WORKER_WATCHING_QUEUES: celery
+    CELERY_BROKER_URL: amqps://xoaonyrj:62WH0JKC8pAZvwyITjX3GpzVg4MMTpNu@albatross.rmq.cloudamqp.com/xoaonyrj
```

Comment:
I know it's a free plan test setup, but should this be better abstracted for the user to change when we provide details?

Reply:
whoops.